content/en/docs/concepts/storage/volumes.md

```diff
@@ -397,6 +397,8 @@ or as read-write, because:
   parts of the cluster.
 * Pods with identical configuration (such as created from a PodTemplate) may
   behave differently on different nodes due to different files on the nodes.
+* Hostpath volume capacity usage is not managed by Kubernetes node/pod ephemeral storage monitoring,
```

Comment:
very trivial nit that I'm fine if you ignore: this line should end in a semicolon or period as these are two separate, but related, complete sentences.